packages/client/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.tsx

```diff
@@ -156,18 +156,14 @@ const TeamTasksHeader = (props: Props) => {
       </TeamHeaderAndAvatars>
       <DashSectionControls>
         {/* Filter by Owner */}
-        <TeamMeta>
```

Comment:
+1 what's the thinking behind this change? Why do we need to update the `TeamTaskHeader` component?